spec/system/admin/tenants_spec.rb

```diff
@@ -40,6 +40,34 @@
       expect(page).to have_current_path root_path
       expect(page).to have_link "Sign in"
     end
+
+    scenario "Updates new tenant default user with the main tenant administrator login credentials" do
+      user = create(:user, password: "secret_password")
+      create(:administrator, user: user)
+      login_as(user)
+
+      visit new_admin_tenant_path
+
+      expect(page).to have_content "When you create a tenant, your current user"
+
+      fill_in "Name", with: "Earthlings"
+      fill_in "Subdomain", with: "earth"
+      click_button "Create tenant"
+
+      expect(page).to have_content "Tenant created successfully"
+
+      click_link "earth.lvh.me"
+
+      expect(page).to have_link "Sign in"
+
+      click_link "Sign in"
```

Comment:
The expectation is redundant since `click_link` expects the link to be there before clicking it. I'm afraid whatever made the test fail wouldn't be solved with this :relieved:, since in the test the link was actually clicked.

```suggestion
      click_link "Sign in"
```